lucas_scheinkerman/README.md

```diff
@@ -1,43 +1,49 @@
-## Differential-drive robots
+# Differential-drive robots
 
-### Definition
-A differential-drive robot (DDR from now) is a robot which has a two-wheel mechanism for allowing it to move. There is a third wheel which prevents the robot from falling and helps it balance, but which has not active role in the movement of the robot. <br>
+## Definition
+
+A differential-drive robot (DDR from now) is a robot which has a two-wheel mechanism for allowing it to move. There is a third wheel which prevents the robot from falling and helps it balance, but which has not active role in the movement of the robot. __
```

Comment:
What are these underscores at the end of the paragraph?

Leave a space after this paragraph. Look [here](https://github.com/ekumenlabs/training_notes_gazebo_ros_2019/blob/e107b9ed91f387df7dcd5dbf60b7fdd0b058013e/lucas_scheinkerman/README.md) how this file looks like. Github takes this as the same paragraph.

Reply:
The same happens in order parts of the document.